web/client/plugins/print/index.js

```diff
@@ -130,6 +159,26 @@ export const standardItems = {
         plugin: Description,
         cfg: {},
         position: 2
+    }, {
+        id: "outputFormat",
+        plugin: OutputFormat,
+        cfg: {
+            "allowedFormats": [
+                {name: "PDF", value: "pdf"},
+                {name: "PNG", value: "png"},
+                {name: "JPEG", value: "jpg"},
+                {name: "GIF", value: "gif"}
```

Comment:
I should not add gif format, it is so unuseful.  ;D

```suggestion
                {name: "JPEG", value: "jpg"}
```